src/forms/registration-popup/index.jsx

```diff
@@ -233,6 +233,12 @@ const RegistrationForm = () => {
     }));
   };
 
+  const handleOnFocus = (e, name) => {
```

Comment:
Why do we need to use `e` param?
It could be:
```suggestion
  const handleOnFocus = (name) => {
```